shell/platform/darwin/ios/framework/Source/FlutterView.mm

```diff
@@ -30,4 +36,87 @@ - (BOOL)enableInputClicksWhenVisible {
   return YES;
 }
 
+void SnapshotRasterizer(ftl::WeakPtr<shell::Rasterizer> rasterizer,
+                        CGContextRef context,
+                        bool is_opaque) {
+  if (!rasterizer) {
```

Comment:
I think the style for this part of the code base is to not use braces for single-line if statements.